config/config.go

```diff
@@ -446,8 +448,46 @@ type ScrapeConfig struct {
 
 	// List of target relabel configurations.
 	RelabelConfigs []*relabel.Config `yaml:"relabel_configs,omitempty"`
+
 	// List of metric relabel configurations.
 	MetricRelabelConfigs []*relabel.Config `yaml:"metric_relabel_configs,omitempty"`
+
+	// List of rules to execute at scrape time.
+	RuleConfigs []*ScrapeRuleConfig `yaml:"scrape_rule_configs,omitempty"`
```

Comment:
Not sure what to do about it, but it will a bit confusing that normal rules are configured using separate files, while scrape-time rules are configured inline. Probably that's fine for now though.